packages/ag-charts-community/src/util/mutex.ts

```diff
@@ -26,7 +26,7 @@ export class Mutex {
     }
 
     public async waitForClearAcquireQueue() {
-        return this.acquire(async () => undefined);
+        return this.acquire(() => Promise.resolve(undefined));
```

Comment:
Why did it care here?

Reply:
There's a new rule for use of `async` keyword, you must use `await` for it to be valid - which seems to make sense generally?